R/pkg/R/functions.R

```diff
@@ -41,9 +41,15 @@ NULL
 #' @param x Column to compute on. In \code{window}, it must be a time Column of
 #'          \code{TimestampType}.
 #' @param format For \code{to_date} and \code{to_timestamp}, it is the string to use to parse
-#'               Column \code{x} to DateType or TimestampType. For \code{trunc}, it is the string
-#'               to use to specify the truncation method. For example, "year", "yyyy", "yy" for
-#'               truncate by year, or "month", "mon", "mm" for truncate by month.
+#'               Column \code{x} to DateType or TimestampType.
+#'
```

Comment:
I'm pretty sure these whitespace will be stripped by roxygen2.
use item if you think that makes sense, otherwise it might be better to keep as a single paragraph

Reply:
Yup, I am fine with fixing. Will check and fix late tomorrow.